Update and improvement in README (#2068)
apache/accumulo: README.md

        <img src="https://img.shields.io/badge/license-ASL-blue.svg" alt="Apache License">
    </a>
</p>

# About

[Apache Accumulo®][accumulo] is a sorted, distributed key/value store that provides robust, scalable data storage and retrieval.

With Apache Accumulo, users can store and manage large
data sets across a cluster. Accumulo uses [Apache Hadoop]'s HDFS to store its data and
[Apache Zookeeper] for consensus.

Visit our 🌐 [project website][accumulo] for news and general information

Download the latest version of Apache Accumulo® [here](https://accumulo.apache.org/downloads/).

# Getting Started

Follow the [quick start] to install and run Accumulo

# Documentation

Read the [Accumulo documentation][docs]

# Examples

* Run the [Accumulo examples][examples] to learn how to write Accumulo clients
* View the [Javadocs][javadocs] to learn the [Accumulo API][api]

More resources can be found on the [project website][accumulo].

# Building

Accumulo uses [Maven] to compile, [test], and package its source.

The following
command :

    mvn package

will build the binary `tar.gz` (`assemble/target/accumulo-<version>-bin.tar.gz`) from source.

> > _Add `-DskipTests` to build without waiting for the tests to run._

Reviewer: This shouldn't be a blockquote. It's not quoting anything/anyone.

limitations under the License.
-->

> <!-- LOGO -->

Reviewer: I don't see any reason to change the the markdown formatting to raw HTML, or change the layout/style of badges or the logo. And, there definitely shouldn't be a change to link to vuejs's site. I assume that was copied/pasted from another project and is a mistake, but I don't think any of this should be changed at all.

Author: > I don't see any reason to change the the markdown formatting to raw HTML, or change the layout/style of badges or the logo. 
 
The main idea is to make it look like this:
![image](https://user-images.githubusercontent.com/23703592/116788862-aa202280-aab4-11eb-8e12-c4f93739098e.png)

The only thing that has been modified is to make it look centered (and with that it would become an HTML format) the rest remains unchanged

---
> And, there definitely shouldn't be a change to link to vuejs's site. I assume that was copied/pasted from another project and is a mistake, but I don't think any of this should be changed at all.

At the moment VueJs does not sponsor Apache ... Although I never know ... 
_This kind of 'slip' is my signature, my apologies!_
It would be a matter of modifying the link .. :)

Reviewer: I think it's better to stick with the Markdown instead of the HTML. I don't think centering is necessarily better or worth it. The Markdown is easier to maintain, and is the convention for many projects on GitHub, especially for the badges.

Author: In my humble opinion regarding maintenance (in this case) there would be no problem because the images are referenced in an external link and therefore it does not matter to use markdown or html.

>

Author: > I don't think centering is necessarily better or worth it.

Honestly, I see it as more minimalist, detailed ... _Hahaha I appeal to democracy ... would there be a way to vote?_

> The Markdown is easier to maintain, .(...) , especially for the badges.

In my humble opinion regarding maintenance (in this case) there would be no problem because the images and badges are referenced in an external link and therefore it does not matter to use markdown or html. Another thing is that there is behind a link automation or something similar.

> and is the convention for many projects on GitHub

There are projects like those of [Facebook](https://github.com/facebook/react-native), [Vue Js](https://github.com/vuejs/vue) that use html in their README

Reviewer: @nicolasalarconrapela We don't just use Markdown because it's shorthand for HTML. We also use it because it's more human-readable. The less HTML, the better. We don't need HTML. This README is also included in the tarball we generate. So, it should be readable in a user's text editor, and the reader should not have to parse HTML/XML tags to read it. Centering the image, in my opinion, does not justify adding HTML that is harder for users to view using a plain text viewer.

Author: Perfect !!! I think it's great !!

<p align="center">
    <a href="https://vuejs.org" target="_blank" rel="noopener noreferrer">
        <img src="contrib/accumulo-logo.png" alt="Vue logo">
    </a>
</p>

<!-- BADGES -->
<p align="center">
    <!-- Build Status -->
    <a href="https://github.com/apache/accumulo/actions">
        <img src="https://github.com/apache/accumulo/workflows/QA/badge.svg" alt="Build Status">
    </a>
    <!-- Maven Central -->
    <a href="https://maven-badges.herokuapp.com/maven-central/org.apache.accumulo/accumulo-core">
        <img src="https://maven-badges.herokuapp.com/maven-central/org.apache.accumulo/accumulo-core/badge.svg"
            alt="Maven Central">
    </a>
    <!-- JavaDocs -->
    <a href="https://www.javadoc.io/doc/org.apache.accumulo/accumulo-core">
        <img src="https://www.javadoc.io/badge/org.apache.accumulo/accumulo-core.svg" alt="JavaDocs">
    </a>
    <!-- Apache License -->
    <a href="https://www.apache.org/licenses/LICENSE-2.0">
        <img src="https://img.shields.io/badge/license-ASL-blue.svg" alt="Apache License">
    </a>
</p>

> # About

Reviewer: I'm opposed to expanding the README to duplicate information on the website. We previously took efforts to substantially minimize the README to make it easier to read, and keep more comprehensive information up-to-date on the website instead. The README really should be minimal, so we don't have to maintain the same information in multiple places. A link to the website should be fine.

Keep in mind that the README is also included in the release tarball, so links to files/pages not in the tarball won't work. That includes everything in `.github/`.

Also, links to specific pages on the website should be minimized. Since this file is included in the release tarball at the time of release, it should endeavor to show information that will not be invalidated by website redesigns or reorganizations.

Author: > I'm opposed to expanding the README to duplicate information on the website. We previously took efforts to substantially minimize the README to make it easier to read, and keep more comprehensive information up-to-date on the website instead. The README really should be minimal, so we don't have to maintain the same information in multiple places. A link to the website should be fine.

I see well that the sections are maintained ... Opinions ... :)

> Keep in mind that the README is also included in the release tarball, so links to files/pages not in the tarball won't work. That includes everything in .github/.
> Also, links to specific pages on the website should be minimized. Since this file is included in the release tarball at the time of release, it should endeavor to show information that will not be invalidated by website redesigns or reorganizations.

It seems good to me I will check the indexing of url's and that they comply with this commitment

[![Build Status][ti]][tl] [![Maven Central][mi]][ml] [![Javadoc][ji]][jl] [![Apache License][li]][ll]

[Apache Accumulo][accumulo] is a sorted, distributed key/value store that provides robust,

> scalable data storage and retrieval. With Apache Accumulo, users can store and manage large data sets across a cluster. Accumulo uses [Apache Hadoop]'s HDFS to store its data and

Reviewer: Since this section doesn't change any wording, we should keep the existing 2 lines, rather than merge them into 1.

> [Apache Zookeeper] for consensus. Check out the [Accumulo project website][accumulo] for
> news and general information.
> 
> [Apache Zookeeper] for consensus.
> 
> Download the latest version of Apache Accumulo® [here][dl].
> 

Reviewer: ```suggestion
[Apache Zookeeper] for consensus.

Download the latest version of Apache Accumulo on the [project website][dl].

```

This command produces `assemble/target/accumulo-<version>-bin.tar.gz`

## Contributing

> Contributions are welcome to all Apache Accumulo repositories
> 
> If you want to contribute, go through our guide [How to contribute](https://accumulo.apache.org/how-to-contribute/)

Reviewer: ```suggestion
Contributions are welcome to all Apache Accumulo repositories.

If you want to contribute, read [our guide on our website][contribute].
```

Can put `[contribute]: https://accumulo.apache.org/how-to-contribute` at the end.

## Export Control

<details>
<summary>Click here to show/hide details</summary>

> <br>

Reviewer: This change can be undone.

[tl]: https://github.com/apache/accumulo/actions
[java-export]: https://www.oracle.com/us/products/export/export-regulations-345813.html
[bouncy-site]: https://bouncycastle.org

> [dl]: https://accumulo.apache.org/downloads/

Reviewer: ```suggestion
[dl]: https://accumulo.apache.org/downloads
[contribute]: https://accumulo.apache.org/how-to-contribute
```

</details>

<br/>

> # License

Reviewer: Do all of these sections really deserve to be `h1` headers? I don't think we need all these sections.
